src/codeu/.idea/codeu.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please see my comment to Jonathan on how to handle IntelliJ and other unrelated files using .gitignore.